### PR TITLE
Remove party validation

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

FLA-1231

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/ai-reviewer-api/src/test/java/ca/bc/gov/open/jag/aireviewerapi/document/validators/documentValidator/ValidateExtractedDocumentTest.java [hunk 1]
```diff
@@ -80,20 +78,6 @@ public void withValidDocumentExecuteWithoutThrowing() throws JsonProcessingExcep
 
         answers.add(documentAnswerField);
 
-        DiligenAnswerField plaintiffAnswerField = DiligenAnswerField.builder()
-                .id(Keys.ANSWER_PLAINTIFF_ID)
```
Comment (KDNeufeld) [inline src/backend/ai-reviewer-api/src/test/java/ca/bc/gov/open/jag/aireviewerapi/document/validators/documentValidator/ValidateExtractedDocumentTest.java:84]: You can also remove the import for this class.  It's now unused.

File: src/backend/ai-reviewer-api/src/test/java/ca/bc/gov/open/jag/aireviewerapi/document/validators/documentValidator/ValidateExtractedDocumentTest.java [hunk 2]
```diff
@@ -185,72 +169,4 @@ public void withRestrictedDocumentThrowException() throws JsonProcessingExceptio
...
-        DiligenAnswerField documentAnswerField = DiligenAnswerField.builder()
-                .id(NOT_DOCUMENT_TYPE_ID)
-                .values(Collections.singletonList(RESPONSE_TO_CIVIL_CLAIM))
-                .create();
-
-        answers.add(documentAnswerField);
-
-        DiligenAnswerField plaintiffAnswerField = DiligenAnswerField.builder()
-                .id(Keys.ANSWER_PLAINTIFF_ID)
-                .values(Arrays.asList(PLAINTIFF, PLAINTIFF))
```
Comment (KDNeufeld) [inline src/backend/ai-reviewer-api/src/test/java/ca/bc/gov/open/jag/aireviewerapi/document/validators/documentValidator/ValidateExtractedDocumentTest.java:202]: You can also remove the import for this class. It's now unused.

File: src/backend/ai-reviewer-api/src/test/java/ca/bc/gov/open/jag/aireviewerapi/document/validators/documentValidator/ValidateExtractedDocumentTest.java (hunk 1 above)
Comment (TayGov) [reply src/backend/ai-reviewer-api/src/test/java/ca/bc/gov/open/jag/aireviewerapi/document/validators/documentValidator/ValidateExtractedDocumentTest.java:84]: Thanks thought I caught all the unused imports

**Review by KDNeufeld — APPROVED** [review]
Great!

**Pull request merged**